app/assets/stylesheets/modules/_article_finder.styl

```diff
@@ -11,6 +11,14 @@
 .fetch-more
   height 3rem
 
+.pageviews 
+  text-align justify
+  direction rtl
+  padding-right: 3rem !important
```

Comment:
Can you find a way to do this without `!important`? (Also, the colon is not necessary here.)